hazelcast/src/main/java/com/hazelcast/internal/blackbox/FieldSensorInput.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.blackbox;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation that can be placed on a field of an object to indicate that it should be tracked by the
+ * Blackbox.
+ */
+@Retention(value = RetentionPolicy.RUNTIME)
+@Target({ElementType.FIELD })
+public @interface FieldSensorInput {
```

Comment:
Key question: do you see anything that would make the Blackbox dangerous to use in production.